### PR TITLE
Fail invalid function file or handler in the function image build phase

### PR DESCRIPTION
**Pull request by rjew**
Validate user given function file and handler during function image build phase to prevent invalid functions reporting as `"READY"`
* `Validator` java program used to load function file and handler to ensure that function is correct
* Clean up source files

### CODE REVIEW

File: function-server/src/main/java/io/dispatchframework/javabaseimage/Dummy.java [hunk 1]
```diff
@@ -1,10 +0,0 @@
-package io.dispatchframework.javabaseimage;
```
Comment (tenczar) [inline function-server/src/main/java/io/dispatchframework/javabaseimage/Dummy.java:1]: Did you mean to delete this class?

**Review by tenczar — COMMENTED** [review]
It looks like there is a lot removed in this pull that isn't related to the validation stuff. Was there a merge/rebase issue?

File: function-server/src/main/java/io/dispatchframework/javabaseimage/Dummy.java (hunk 1 above)
Comment (rjew) [reply function-server/src/main/java/io/dispatchframework/javabaseimage/Dummy.java:1]: There's just a lot of unused code that wasn't removed in the source dir update. That Dummy class doesn't seem to be used.

Comment (tenczar) [reply function-server/src/main/java/io/dispatchframework/javabaseimage/Dummy.java:1]: Ok, that's fine with me.

**Review by tenczar — APPROVED** [review]

**Pull request merged**